content/_index.en.md

```diff
@@ -2,6 +2,7 @@
 title: Open Terms Archive
 html_description: Open Terms Archive publicly records every version of the terms of digital services to enable democratic oversight.
 hero:
+  anchor: mission-statement
```

Comment:
Why is the anchor different in French and in English? 🤔

Reply:
Because it is translated.

Reply:
I find this weird. “mission” is the same in French and in English, and having the same anchor means having the same document structure and enables easier linking across languages.